### ISSUE TITLE
Bot that downloads photos from an (encrypted) group chat and stores them in a folder 

### ISSUE THREAD

**Issue by arthurlutz** [matrix-org/go-neb#250]
Follow up of issue posted on the "python version" of the bot https://github.com/matrix-org/Matrix-NEB/issues/23

I was wondering if this bot project would be a good starting point to write a bot that sits in an encrypted group chat and for every photo / video that is posted there, stores them to disk (which behind would be synced using syncthing, shared by members using syncting and served over the web by a nextcloud instance). This is for a family usecase as an alternative to WhatsApp and I realised some members were loosing time saving all the photos posted in the group chat one by one. 

Thanks for you input and pointers.

**Comment by t3chguy** [comment]
No it would not as it does not support encryption whereas the python one does

**Comment by uhoreg** [comment]
Of course, if anyone wants to implement encryption for this, please feel free to do so. :)

(And FTR, the Python SDK does not support encryption yet -- the PRs are still being reviewed. slowly...)
